File: Scripts/Models/StatusLog.js
const StatusLog = class {
    constructor (
        id
        , taskId
        , status_id
        , dateTime
    ) {
        this.id = id
        this.taskId = taskId
        this.statusId = status_id
        this.dateTime = dateTime
    }
}

module.exports = { StatusLog }

File: ProTracker.Backend/ProTracker.Models/StatusLog.cs
namespace ProTracker.Models;

public class StatusLog
{
    public int Id { get; set; }
    public int TaskId { get; set; }
    public int StatusId { get; set; }
    public long DateTime { get; set; }

    // Navigation properties
    public TrackerTask Task { get; set; } = null!;
    public Status Status { get; set; } = null!;
}
